src/main.ts

```diff
@@ -334,30 +382,17 @@ export default class NewZettel extends Plugin {
             : selectionPos.anchor.line < selectionPos.head.line; // else they are not on the same line and just check if anchor is before head
 
         // if anchorCorrect use as is, else switch
-        const virtualAnchor = anchorCorrect
-          ? selectionPos.anchor
-          : selectionPos.head;
-        const virtualHead = anchorCorrect
-          ? selectionPos.head
-          : selectionPos.anchor;
-        editor!.replaceRange(
-          " ".repeat(spaceBefore) + newLink(title) + " ".repeat(spaceAfter),
-          virtualAnchor,
-          virtualHead
-        );
-        this.makeNote(nextPath(title), title, fileLink, true, openNewFile);
+        const virtualAnchor = anchorCorrect ? selectionPos.anchor : selectionPos.head;
+        const virtualHead = anchorCorrect ? selectionPos.head : selectionPos.anchor;
+        // editor!.replaceRange(" ".repeat(spaceBefore) + newLink(title) + " ".repeat(spaceAfter), virtualAnchor, virtualHead);
```

Comment:
Why was this moved into the call back? I could be miss remembering this but doesn't that call back call on the new file not the old one? If not please let me know I could very well be wrong and am trying to get through all the PRs before bed. (Aiming for a working next version release on or before friday)

Reply:
It was moved into a callback because it placed replace a selection/placed with a link to the new note, but if were to be aborted because of wrong setting etc. it shouldn't make a link to a non-existing file aka. edit the old one.